### PR TITLE
Reword comments and fix typo in buildcontext.go

### PR DESCRIPTION
**Pull request by joycecodes**
Non-functional change.

### CODE REVIEW

File: pkg/build/nodeimage/buildcontext.go [hunk 1]
```diff
@@ -125,7 +125,7 @@ func (c *buildContext) buildImage(bits kube.Bits) error {
 		return err
 	}
 
-	// pre-pull images that were not part of the build and write CNI / storage
+	// pre-pull images that were not part of the build and write CNI / Storage
```
Comment (stmcginnis) [inline pkg/build/nodeimage/buildcontext.go:128]: Storage isn't a proper name here, I don't think it needs to be capitalized.

File: pkg/build/nodeimage/buildcontext.go [hunk 2]
```diff
@@ -249,7 +249,7 @@ func (c *buildContext) prePullImagesAndWriteManifests(bits kube.Bits, parsedVers
 		c.logger.Errorf("Image build Failed! Failed write default Storage Manifest: %v", err)
 		return nil, err
 	}
-	// all builds should install the default storage driver images currently
+	// all builds should install the default Storage driver images currently
```
Comment (stmcginnis) [inline pkg/build/nodeimage/buildcontext.go:252]: Same here.

**Review by stmcginnis — COMMENTED** [review]
Some slight improvements here, but nothing I would deem too important. But we will need the EasyCLA issues sorted before we can accept this. Please address the issues pointed out by the comment from EasyCLA and let us know if anything doesn't make sense.

**Comment by kundan2707** [comment]
@joycecodes  please sign cncf-cla

File: pkg/build/nodeimage/buildcontext.go (hunk 1 above)
Comment (BenTheElder) [reply pkg/build/nodeimage/buildcontext.go:128]: CNI is only capitalized because it is an acronym AND a proper name. In this case storage is neither.

**Comment by joycecodes** [comment]
> It looks like maybe your second commit is OK with easycla but not your first one. It would be good to squash commits anyway.
> 
> You can squash you commits and update it to be the correct author using something like the following:
> 
> ```shell
> git rebase -i HEAD~2
> # change the second commit to squash, fix up combined commit message
> git commit --amend --author="First Last <email@example.com>" --no-edit
> git push -f origin comments
> ```
> 
> We need to make sure changes are covered by the CLA before we can consider accepting them.

Thank you for your patience!

File: pkg/build/nodeimage/buildcontext.go [hunk 3]
```diff
@@ -327,19 +327,18 @@ func (c *buildContext) prePullImagesAndWriteManifests(bits kube.Bits, parsedVers
...
 	runArgs := []string{
-		"-d", // make the client exit while the container continues to run
-		// the container should hang forever, so we can exec in it
-		"--entrypoint=sleep",
+		"-d",                 // make the client exit while the container continues to run
+		"--entrypoint=sleep", // the container should hang forever, so we can exec in it
 		"--name=" + id,
 		"--platform=" + dockerBuildOsAndArch(c.arch),
-		"--security-opt", "seccomp=unconfined", // ignore seccomp
+		"--security-opt", "seccomp=unconfined", // disable secure computing mode
```
Comment (BenTheElder) [inline pkg/build/nodeimage/buildcontext.go:341]: This change is wrong, seccomp is a specific technology, which we're ignoring by setting this container to unconfined. The old comment is not very detailed, but the new comment is inaccurate. If the reader doesn't know what seccomp is, they should not depend on an inline explanation though, it's rather complicated.

**Review by BenTheElder — COMMENTED** [review]
one comment, the rest looks good!

File: pkg/build/nodeimage/buildcontext.go (hunk 3 above)
Comment (BenTheElder) [reply pkg/build/nodeimage/buildcontext.go:341]: I think we should just drop this comment, it's not helpful.

Comment (joycecodes) [reply pkg/build/nodeimage/buildcontext.go]: Makes sense! Thanks for the catch :)

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**